src/main/java/com/ait/lienzo/client/core/shape/wires/picker/ColorMapBackedPicker.java

```diff
@@ -118,10 +116,10 @@ public PickerPart findShapeAt(int x, int y)
             y = (int) Math.round(temp.getY());
         }
 
-        String color = BackingColorMapUtils.findColorAtPoint(m_imageData, x, y);
+        ImageDataPixelColor color = m_ctx.getImageDataPixelColor(x, y);
```

Comment:
It looks like after `color = Context2D#getImageDataPixelColor(x, y);` always follow `color.toBrowserRGB` with one small exception - in one place `return null if color.A is not 255 else color.toBrowserRGB`;

Do we need to have `ImageDataPixelColor` class and create it's objects?